pkg/apis/servicecatalog/v1beta1/types.go

```diff
@@ -66,6 +69,7 @@ type ClusterServiceBrokerSpec struct {
 
 	// RelistBehavior specifies the type of relist behavior the catalog should
 	// exhibit when relisting ClusterServiceClasses available from a broker.
+	// +optional
```

Comment:
`RelistBehavior` is not optional.

Reply:
Hmm, the API server currently accepts brokers without it without complaining. Also, the ups-broker example doesn't include it. 

And, IMHO, this really should be optional. After all, it can be defaulted to something reasonable (like it is now - Duration/15 minutes). Why would we want to force users to deal with this? It would be like forcing them to specify `imagePullPolicy` when creating pods.

Reply:
You are right. I was looking at this portion of the validation code to see that relistBehavior is required.

https://github.com/kubernetes-incubator/service-catalog/blob/31bbf5539dd32d953bc777e01ce44c66a9c4ca27/pkg/apis/servicecatalog/validation/broker.go#L98-L102

And didn't see that defaults are being established here.

https://github.com/kubernetes-incubator/service-catalog/blob/31bbf5539dd32d953bc777e01ce44c66a9c4ca27/pkg/apis/servicecatalog/v1beta1/defaults.go#L31-L33